docs/privacy.md

```diff
@@ -0,0 +1,61 @@
+- `storage`: we use it to keep user data on the phone between app uses, so that the user doesn't need to login every time they open the app, or see the intro screens again. We also keep the user's language selection for instance.
+- `Cloud messaging`: for remote notifications. As I write this document, cloud messaging has been disabled from the app. It was active until v2.0.11.
+
+MapSwipe does not use any advertising features.
+
+## Sentry
+
+Debugging a mobile app once released to an app store is almost impossible without the app sending back some sort of technical information when it hits a problems. We use a service called [Sentry](sentry.io) which sends information from the app when there's a bug in it.
+
+The information it collects includes:
+- a user Id, which we don't link to any other information (meaning we as developers have no idea who encountered the bug, which is often frustrating when trying to fix them)
+- the phone model
+- the ios or android version used by the phone
+- the version of MapSwipe used
+- the type of network connection used (wifi/4g...)
+- the language setting of the phone
+- the screen size
+- etc...
+
+We do not share this information, and only access it if a bug is problematic enough that we need details to understand when it happens.
+
+## Technical details
+
+This section is mostly for internal use.
+
+### Firebase analytics
+
+Details on how it collects data on [google](https://support.google.com/analytics/answer/10285841) and [firebase](https://firebase.google.com/docs/ios/app-store-data-collection) docs.
+
+#### iOS
+
+In short:
+- firebase assigns a unique ID to each app instance (so 1 MapSwipe on 1 phone), called `app instance id`, but it's not linked to any user identifier. We can disable collection of this ID in analytics, which means we would not be able to understand a user's journey through the app.
+- Apple's location service is not used, country is derived from an anonymized IP address
+- As we don't collect IDFA, we do not have access to age, gender, interests of the user ([source](https://support.google.com/firebase/answer/9268042?visit_id=637626397084989644-2548047988&rd=1))
+- We removed IDFV [collection](https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable_idfv_collection) in analytics. IDFV is an apple identifier that seems to uniquely identifier a device, and would allow [tracking usage of various apps of the same provider](https://developer.apple.com/forums/thread/659416) (for instance, track an iOS user across BRC's apps). We don't need this.
+- Analytics data is not used for [personalised advertising](https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable_personalized_advertising_features), although we never linked an ad account to the analytics account for MapSwipe.
```

Comment:
Would remove 'although' as it implies we once did. Use full stop instead:
Analytics data is not used for [personalised advertising](https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable_personalized_advertising_features). We have never linked an ad account to the analytics account for MapSwipe.